internal/cmd/manager/instance/initdb/cmd.go

```diff
@@ -109,6 +113,9 @@ func NewCmd() *cobra.Command {
 		"executed inside application database right after the database is created")
 	cmd.Flags().StringVar(&postInitTemplateSQLStr, "post-init-template-sql", "", "The list of SQL queries to be "+
 		"executed inside template1 database to configure the new instance")
+	cmd.Flags().StringVar(&postInitApplicationSQLRefsFolder, "post-init-application-sql-refs-folder",
```

Comment:
does it has access to secrets or configmaps, if so, it would easy, we even don't need to keep them in a folder, but if we want to keep them in a folder, anyway we need to know the folder name, except hardcoding the path, i cannot think of other ideas.

moreover the value of this option could be used to control the the logic
```go
func (info InitInfo) executePostInitApplicationSQLRefs(sqlUser *sql.DB) error {
	if info.PostInitApplicationSQLRefsFolder == "" {
		return nil
	}
        # ....
}
```